operator/main.go

```diff
@@ -162,7 +164,10 @@ func main() {
 			externalScalerCfg,
 		)
 	})
-
+	setupLog.Info(fmt.Sprintf("Operator Version: %s", build.Version()))
+	setupLog.Info(fmt.Sprintf("Operator Commit: %s", build.GitCommit()))
+	setupLog.Info(fmt.Sprintf("Go Version: %s", goruntime.Version()))
+	setupLog.Info(fmt.Sprintf("Go OS/Arch: %s/%s", goruntime.GOOS, goruntime.GOARCH))
```

Comment:
Can we move this to a shared function or not?

Reply:
yeah, I have moved it to the build pkg (where the version and commit values are)